### DISCUSSION TITLE
Hovers with `<details>` sort of work, but then they don't

### DISCUSSION THREAD

**Discussion by PEZ** [microsoft/vscode-discussions#339] [Extension Development QnA] [answered]
I tried to use a `<details><summary>` thing in a hover. This used to be sanitized away, but now they are legal tags. However, then it behaves weirdly. Sometimes it immediately collapses when I expand it, and even when it stays open, there's no scroll appearing, and if I try to scroll the hover closes.

![6E98718B-686F-4886-84E2-5B1E057774D2-11665-0001DAAB870DF6F8](https://user-images.githubusercontent.com/30010/209222004-5b8815bc-1dc4-44a9-933c-68852bacc7a3.gif)

Is this something worth reporting an issue about?

**Comment by burkeholland** [comment, marked as answer]
I couldn't find a definitive list of supported HTML tags, but this does look like a good candidate for an issue considering it was rendered in the first place.
